charts/promitor-agent-scraper/templates/clusterrolebinding.yaml

```
@@ -14,6 +14,10 @@ roleRef:
   name: {{ template "promitor-agent-scraper.fullname" . }}
 subjects:
   - kind: ServiceAccount
-    name: {{ template "promitor-agent-scraper.serviceAccountName" . }}
+    {{- if .Values.rbac.serviceAccount.create }}
```

In deployment we use this, should we align? (not RBAC expert)
```suggestion
    {{- if .Values.rbac.create }}
    {{- if .Values.rbac.serviceAccount.create }}
```